### DISCUSSION TITLE
Incorrect output when using OVER time

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#238] [Q&A] [answered]
Hi @alex-karpovich ,
We are using "OVER" for rolling up data at every 1 ms.
The query is as below
`select last{} (price) from Test over TIME (1ms) where timestamp BETWEEN 1661419341903 AND 1661419342222 AND SYMBOL IN ('ABC'`)

The above query works fine when records are present in the "Test" stream for the give timestamp range.
However, when there is NO records present in the "Test" stream for the specified time range, we still get 2 records with rolled over timestamp with NULL value for "price" column. 
If we execute the above query as normal select query by removing "OVER", then it works as expected with NO records in output. 

Output in shell
```
>_,TIMESTAMP,SYMBOL,TYPE,price
0,2022-08-25 09:22:23.110,,_
1,2022-08-25 09:22:25.521,,_

```

Expected output is "NO records" instead of the 2 records showing up. 

NOTE: The data is stored using "spaces" and the query is executed in timebase shell.

**Comment by alex-karpovich** [comment, marked as answer]
Greeting @ShivaniSK , please try 6.1.14 TimeBase server

**Reply by ShivaniSK** [reply]
Thanks @alex-karpovich
